File: ADS 2024.1/POO/atividades_enviar/exercicio3/q04.js
function arrayParaString(numeros) {
    var resultado = "";
    numeros.forEach(function (num, indice) {
        resultado += num.toString();
        if (indice < numeros.length - 1) {
            resultado += "-";
        }
    });
    return resultado;
}
// Usando a função
var meuArray = [1, 2, 3, 4, 5];
var stringResultado = arrayParaString(meuArray);
console.log(stringResultado); // Saída: 1-2-3-4-5

File: ADS 2024.1/POO/atividades_enviar/exercicio3/q04.java
public static String arrayParaString(int[] numeros) {
    StringBuilder resultado = new StringBuilder();

    for (int i = 0; i < numeros.length; i++) {
        resultado.append(numeros[i]);
        if (i < numeros.length - 1) {
            resultado.append("-");
        }
    }

    return resultado.toString();
}

public static void main(String[] args) {
    int[] meuArray = {1, 2, 3, 4, 5};
    String stringResultado = arrayParaString(meuArray);
    System.out.println(stringResultado); // Saída: 1-2-3-4-5
}